roles/edpm_update/molecule/default/prepare.yml

```diff
@@ -17,5 +17,54 @@
 
 - name: Prepare
   hosts: all
-  roles:
```

Comment:
there should be separate molecule scenarios for

1) do nothing (ovs not updated)
2) ovs updated but not reloaded
3) ovs updated and reloaded

I think these tests should belong to edpm_ovs role itself (not to edpm_update). edpm_update role may want to include the tasks from edpm_ovs (maybe), but it should be possible to execute the `edpm_ovs` role independently, I think.